setup.cfg

```
@@ -11,8 +11,9 @@ license_file = LICENSE
 keywords = physics fitting numpy scipy tensorflow pytorch jax
 project_urls =
     Documentation = https://pyhf.readthedocs.io/
-    Source = https://github.com/scikit-hep/pyhf
-    Tracker = https://github.com/scikit-hep/pyhf/issues
+    Source Code = https://github.com/scikit-hep/pyhf
+    Issue Tracker = https://github.com/scikit-hep/pyhf/issues
+    Release Notes = https://pyhf.readthedocs.io/en/v0.6.2/release-notes.html
```

I don't know of any way to get this URL to not be version dependent unless we use `stable`

```
https://pyhf.readthedocs.io/en/stable/release-notes.html
```

but I know that @kratsg wants to avoid using that. If we keep the version number in the URL we need to add `setup.cfg` back to `bump2version` control.

---

oh man, you beat me to it.

---

@kratsg As you requested changes in the review, do you want this to be 
```suggestion
    Release Notes = https://pyhf.readthedocs.io/en/stable/release-notes.html
```
and then remove `setup.cfg` from `.bumpversion.cfg`?

---

Yep. I think fundamentally, we'd want to ensure that we're not changing these links so much. Given all previous version releases are listed on that page from `stable`, I think it's fine if we point to stable.

---

Oh also, because the url for the pypi page is generally pointing to latest stable release anyway, it makes sense that we use the stable url.

---

Great! SGTM.